Applications/SlicerApp/qSlicerAppMainWindow.cxx

```diff
@@ -51,91 +52,96 @@ void setThemeIcon(QAction* action, const QString& name)
-{
-}
+{}
 
 //-----------------------------------------------------------------------------
 qSlicerAppMainWindowPrivate::~qSlicerAppMainWindowPrivate() = default;
 
 //-----------------------------------------------------------------------------
-void qSlicerAppMainWindowPrivate::init()
+void
+qSlicerAppMainWindowPrivate::init()
 {
   Q_Q(qSlicerMainWindow);
   this->Superclass::init();
 }
 
 //-----------------------------------------------------------------------------
-void qSlicerAppMainWindowPrivate::setupUi(QMainWindow * mainWindow)
+void
+qSlicerAppMainWindowPrivate::setupUi(QMainWindow * mainWindow)
 {
   //----------------------------------------------------------------------------
   // Add actions
   //----------------------------------------------------------------------------
-  QAction* helpKeyboardShortcutsAction = new QAction(mainWindow);
+  QAction * helpKeyboardShortcutsAction = new QAction(mainWindow);
   helpKeyboardShortcutsAction->setObjectName("HelpKeyboardShortcutsAction");
   helpKeyboardShortcutsAction->setText(qSlicerAppMainWindow::tr("&Keyboard Shortcuts Reference"));
-  helpKeyboardShortcutsAction->setToolTip(qSlicerAppMainWindow::tr("Raise a window that lists commonly-used keyboard shortcuts."));
+  helpKeyboardShortcutsAction->setToolTip(
+    qSlicerAppMainWindow::tr("Raise a window that lists commonly-used keyboard shortcuts."));
 
-  QAction* helpDocumentationAction = new QAction(mainWindow);
+  QAction * helpDocumentationAction = new QAction(mainWindow);
   helpDocumentationAction->setObjectName("HelpDocumentationAction");
   helpDocumentationAction->setText(qSlicerAppMainWindow::tr("Documentation"));
   helpDocumentationAction->setShortcut(QKeySequence(qSlicerAppMainWindow::tr("Ctrl+1", "Documentation")));
 
-  QAction* helpQuickStartAction = new QAction(mainWindow);
+  QAction * helpQuickStartAction = new QAction(mainWindow);
   helpQuickStartAction->setObjectName("HelpQuickStartAction");
   helpQuickStartAction->setText(qSlicerAppMainWindow::tr("Quick Start"));
 
-  QAction* helpGetHelpAction = new QAction(mainWindow);
+  QAction * helpGetHelpAction = new QAction(mainWindow);
   helpGetHelpAction->setObjectName("HelpGetHelpAction");
   helpGetHelpAction->setText(qSlicerAppMainWindow::tr("Get Help"));
 
-  QAction* helpUserInterfaceAction = new QAction(mainWindow);
+  QAction * helpUserInterfaceAction = new QAction(mainWindow);
   helpUserInterfaceAction->setObjectName("HelpUserInterfaceAction");
   helpUserInterfaceAction->setText(qSlicerAppMainWindow::tr("User Interface"));
 
-  QAction* helpVisitSlicerForumAction = new QAction(mainWindow);
+  QAction * helpVisitSlicerForumAction = new QAction(mainWindow);
   helpVisitSlicerForumAction->setObjectName("HelpVisitSlicerForumAction");
   helpVisitSlicerForumAction->setText(qSlicerAppMainWindow::tr("Visit the Slicer Forum"));
 
-  QAction* helpBrowseTutorialsAction = new QAction(mainWindow);
+  QAction * helpBrowseTutorialsAction = new QAction(mainWindow);
```

Comment:
Let's keep the current style. How adding a space before the * would be any improvement?

Reply:
Converted to